packages/duplicate-package-checker-webpack-plugin/README.md

```diff
@@ -75,6 +75,10 @@ Packages with different major versions introduce backward incompatible changes a
 
 It is suggested that strict mode is kept enabled since this improves visibility into your bundle and can help in solving and identifying potential issues.
 
+## Node version support
+
+This package was developed and tested using Node 10 up to Node 14. This package has not been thoroughly tested using Node 16 or greater. No known issues have been reported in newer Node versions, so this package is set to allow Node version 10 or greater. However, consumers utilizing Node 16 or greater should use this package at their own risk.
```

Comment:
```suggestion
This package was developed and tested using Node 10 up to Node 14. Consumers using Node 16 or greater are advised to use it at their own risk since those versions are not officially supported due to lack of thorough testing. 
```

Reply:
Updated all packages: https://github.com/cerner/terra-toolkit/pull/731/commits/3ad59b5595d014de374fa5b0e8b5e46eac7a87fc